tests/cxx/iwyu_stricter_than_cpp.cc

```diff
@@ -263,6 +263,36 @@ void TestAutocast() {
       6, 7, 8, 9,
       // IWYU: TplIndirectStruct2 is...*iwyu_stricter_than_cpp-i2.h
       10);
+
+  TplFnValues(
+      // IWYU: TplIndirectStruct3 is...*iwyu_stricter_than_cpp-i5.h
+      1,  // onlyArgumentTypeProvided
```

Comment:
We use a tricky named-arguments style in IWYU (derived from LLVM, I think), like here: https://github.com/include-what-you-use/include-what-you-use/blob/master/iwyu_ast_util.cc#L433.

Would probably be nice to use that here as well, e.g. 
```
TplFnValues(
  // IWYU...
  /*onlyArgumentTypeProvided=*/1,
  // IWYU...
  /*allForwardDeclared=*/2,
  ...